src/control-flow/cfg-typescript.ts

```diff
@@ -319,3 +323,82 @@ function processTryStatement(trySyntax: SyntaxNode, ctx: Context): BasicBlock {
     });
   });
 }
+
+const nodeType = {
+  // function‑declaration cases
+  functionDeclaration: "function_declaration",
+  generatorFunctionDeclaration: "generator_function_declaration",
+
+  // inline/function‑expression cases
+  generatorFunction: "generator_function",
+  arrowFunction: "arrow_function",
+  functionExpression: "function_expression",
+
+  // methods
+  methodDefinition: "method_definition",
+
+  // identifier lookups
+  identifier: "identifier",
+  propertyIdentifier: "property_identifier",
+
+  // Unnamed functions
+  anonymous: "<anonymous>",
+};
+
+const variableDeclaratorQueryAndTag = {
+  query: `
+    (lexical_declaration
```

Comment:
What of `var` definitions, and of definitions without either `var`, `let` or `const`?